core/build.gradle.kts

```diff
@@ -50,14 +51,18 @@ dependencies {
     testImplementation("io.grpc:grpc-inprocess")
 
     testImplementation("com.exactpro.th2:junit-jupiter-integration:0.0.1")
-    testImplementation("org.awaitility:awaitility:4.2.1")
 
     testRuntimeOnly("com.mysql:mysql-connector-j:8.3.0") {
         because("mysql support")
     }
     testRuntimeOnly("com.oracle.database.jdbc:ojdbc11:23.3.0.23.09") {
         because("oracle support")
     }
+
+    testFixturesImplementation("org.junit.jupiter:junit-jupiter:5.10.0")
+    testFixturesImplementation(platform("org.testcontainers:testcontainers-bom:1.19.7"))
+    testFixturesImplementation("org.testcontainers:testcontainers")
+    testFixturesImplementation("org.testcontainers:oracle-xe")
```

Comment:
issue: This is not what the test fixture is supposed to be used. It must not have dependencies on any specific test framework. It just contains code (like test-friendly implementation of interfaces or some test data) that can be reused among tests.
If you don't want to repeat the dependencies - extract them. You can use the libs catalog to create bundle with dependencies and use it in all projects where they are needed